v201705/ad_group_ad.go

```diff
@@ -327,10 +334,11 @@ func (s *AdGroupAdService) Mutate(adGroupAdOperations AdGroupAdOperations) (adGr
 	operations := []adGroupAdOperation{}
 	for action, adGroupAds := range adGroupAdOperations {
 		for _, adGroupAd := range adGroupAds {
+			ad := []interface{}{adGroupAd}
 			operations = append(operations,
 				adGroupAdOperation{
 					Action:    action,
-					AdGroupAd: []interface{}{adGroupAd},
```

Comment:
Why make this change?

Reply:
This is in relation to the previous change/comment